engineering-feedback/readme.md

```diff
@@ -24,7 +26,16 @@ Good engineering feedback provides enough information for those who are not part
 
 ### Examples of Good Engineering Feedback
 
-* Coming soon
+Include details about the scenario and verbatim as much as possible in the main content.
+
+For the executive summary here is an evolution of transforming a fictitious feedback with the above high-quality engineering feedback guidance in mind:
+
+* Initial feedback - Emailing directly from Teams is difficult

```

Comment:
This is great way to demonstrate the 3 properties of good feedback. Just a couple of thoughts:
* Would it make sense to make it into a table with 2 columns instead of bullets to make it easier to see the "Original", "Making it Goal Oriented", "Adding Specifics", "Making it Actionable"?
* Would we want to try using one of the real world examples like below?

| Stage                            | Feedback Evolution                    |
| -------------------------------- | ------------------------------------- |
| Initial feedback                 | Azure Functions Service Bus Trigger is slow. |
| Making it **Goal Oriented**      | Customer scenario was to **use Azure Functions to process many messages per second with minimum latency and in a given order.** |
| Adding **Specifics**             | Customer scenario was to receive a **total of 250 messages/second from 50 producers with requirement for ordering & minimum latency, using a Service Bus topic with sessions enabled for ordering. Batch receiving is not supported in Azure Functions Service Bus Trigger.** |
| Making it **Actionable**         | Customer scenario was to receive a total of 250 messages/second from 50 producers with requirement for ordering & minimum latency, using a Service Bus topic with sessions enabled for ordering. **According to [Microsoft documentation](https://github.com/Azure/azure-functions-servicebus-extension/issues/15), batch receiving is recommended for better performance but this is not currently supported in Azure Functions Service Bus Trigger. The impact and workaround was choosing containers over Functions. The desired outcome is for Azure Functions to support Service Bus sessions with batch and non-batch processing for all Azure Functions GA languages.** |

Reply:
I like the table idea, makes sense; I liked the canonical fictitious example as it was created by the CSE Feeedback team and their take on the evolution, does that one feel wrong to you? Just wondering why we'd change from it to this (although this looks good too). Worth discussing with @oosinubi

Reply:
Turned into table. Can we decide on which example to use in a future PR?

Reply:
@nzthiago - I think the "Teams example" was not from CSE Feedback team but from a different group that receives feedback from end-users about all kinds of Microsoft products. Since our audience are developers/engineers, I thought that it maybe better to provide a canonical example using a developer example of feedback (versus the Team example that may sound like a business-user feedback). Thoughts?

Reply:
It was the actual example used in the video from @oosinubi related to our Engineering Feedback, so if that was not a good example they might need to update there too. Let's discuss, I'm happy with either.